### PR TITLE
Added Kali Linux Cheatsheet

### PR DESCRIPTION
**Pull request by MathewtheCoder**
###### What does your Pull Request do (check all that apply)?

- [x] New Instant Answer
    - [x] Cheat Sheets: **`New Kali Linux Cheat Sheet`**
    
**'Cheatsheet for Kali Linux Commands`**

Provide an overview of the changes this pull request introduces.

Instant Answer Page: https://duck.co/ia/view/kali_linux_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@MathewtheCoder Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please head to https://quackslack.herokuapp.com/ to get an invite.

>More Info: http://docs.duckduckhack.com/submitting/submitting-overview.html

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/kali_linux.json [hunk 1]
```diff
@@ -0,0 +1,286 @@
+{
+    "id": "kali_linux_cheat_sheet",
+    "name": "KaliLinux",
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/kali_linux.json]: Lets change name to "Kali Linux"

File: share/goodie/cheat_sheets/json/kali_linux.json [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+    "description": "Cheatsheet of Kali Linux Commands",
+
+    "metadata": {
+        "sourceName": "NullByte",
+        "sourceUrl" : "http://null-byte.wonderhowto.com/how-to/hack-like-pro-ultimate-command-cheat-sheet-for-metasploits-meterpreter-0149146/"
+    },
+
+    "aliases": [
+        "kali", "kali linux", "kali linux commandline"
```
Comment (jophab) [inline share/goodie/cheat_sheets/json/kali_linux.json]: remove' kali linux ' since its same as your filename

**Comment by jophab** [comment]
make "kali-linux.json" the file name.
We have to use  hyphen between words in file name.

**Comment by jophab** [comment]
you can close the new pr you made for kali

File: share/goodie/cheat_sheets/json/kali-linux.json [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+    ],
+
+    "template_type": "terminal",
+
+    "section_order": [
+        "Basic Commands",
+        "File System Commands",
+        "Networking Commands",
+        "System Commands",
+        "UserInterface Commands",
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/kali-linux.json]: Need a space between User and Interface here

File: share/goodie/cheat_sheets/json/kali-linux.json [hunk 4]
```diff
@@ -0,0 +1,286 @@
+{
+    "id": "kali_linux_cheat_sheet",
+    "name": "Kali Linux",
+    "description": "Cheatsheet of Kali Linux Commands",
```
Comment (gaulrobe) [inline share/goodie/cheat_sheets/json/kali-linux.json]: Do we want to put `Cheatsheet` in the description? Or maybe just keep this as `Kali Linux Commands`?

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Kali Linux Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/kali_linux_cheat_sheet_mobile.png?KFowQg0HxD)

**Comment by alohaas** [comment]
@MathewtheCoder this LGTM. Are there other changes you wish to make? Otherwise, I'll merge 👍

**Comment by MathewtheCoder** [comment]
I made all the necessary changes.You can merge.Thank u

**Comment by daxtheduck** [comment]
@MathewtheCoder this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by alohaas** [comment]
@MathewtheCoder Thanks again for this new IA! As @daxtheduck explained above, it should go live soon.

If you have other IA ideas, other members of the community or I would be happy to discuss them with you on Slack. Happy hacking!

**Pull request merged**